plasmapy/classes/tests/test_Plasma.py

```diff
@@ -99,6 +99,16 @@ def test_Plasma3D_derived_vars():
 
 class Test_PlasmaBlobRegimes():
     def test_intermediate_and_both_energies(self):
+        r"""
+        Method to test Gamma and Theta values on PlasmaBlob.regimes()
+
+        Tests that input values return an intermediate coupling regime
```

Comment:
I think this sentence is mixing two different regime parameters. Ask I discussed on the comment about the state space diagram, coupling and degeneracy are two different concepts (even though they both tend to appear at higher densities).